frontend/pages/hosts/details/DeviceUserPage/ManualEnrollMdmModal/ManualEnrollMdmModal.tsx

```diff
@@ -11,17 +11,17 @@ import Spinner from "components/Spinner";
 
 import mdmAPI from "services/entities/mdm";
 
-export interface IInfoModalProps {
+interface IManualEnrollMdmModalProps {
   onCancel: () => void;
-  token: string;
+  token?: string;
 }
 
-const baseClass = "manual-enroll-mdm-modal";
+const baseClass = "manual-enroll-mdm-modal enroll-mdm-modal";
 
-const ManualEnrollMdmModal = ({
+const EnrollMdmModal = ({
```

Comment:
Let's leave this as `ManualEnrollMdmModal` since the other one is `AutoEnrollMdmModal`?

Reply:
Guess I missed this name change in both. Thanks.